braindecode/classifier.py

```diff
@@ -268,3 +270,17 @@ def predict(self, X):
 
         """
         return self.predict_proba(X).argmax(1)
+
+    def predict_trials(self, X):
+        """Return trialwise predictions and targets.
```

Comment:
Great, can you add one sentence how the expected output shape will be for trial_predictions and trial_labels, including meaning of dimensions for trial_predictions. That should be helpful